app/assets/javascripts/config.js.coffee.erb

```diff
@@ -19,7 +19,8 @@
     csrfToken: csrfToken
     hoverTimeoutMs: 0
     gibsUrl: "/assets/gibs-example.jpeg?{timeparam}SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER={product}&STYLE=&TILEMATRIXSET={projection}_{resolution}&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}&FORMAT=image%2F{format}{extra}"
-    browseScalerUrl: "/assets/gibs-example.jpeg#"
+    granuleBrowseScalerUrl: "/assets/gibs-example.jpeg#"
+    collectionBrowseScalerUrl: "/assets/gibs-example.jpeg#"
     present: -> Date.UTC(2014, 2, 1)
```

Comment:
Patrick, do you think we should combine these two links into one?
And in coffeescripts we can say "#{scalerUrl}/datasets/C123-ABC?foo=bar" and "#{scalerUrl}/granules/G123-ABC?foo=bar"?